src/components/home/statsgrid.tsx

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+
+const StatsGrid: React.FC = () => {
+  return (
+    <div className="mx-auto grid max-w-4xl grid-cols-4 divide-x divide-gray-300 text-center">
```

Comment:
```suggestion
    <div className="mx-auto grid grid-cols-4 text-center">
```
make the text larger here instead of restricting width